oci/private/versions.bzl

```diff
@@ -1,163 +1,25 @@
-        "linux_i386": "sha256-OIpcJTWqXgTwi6wg3IG0/C3S03oSFkFyB9kYtKYFhHM=",
-        "linux_x86_64": "sha256-9LqZadf5mRiMRef4p+xOWDsc8R9HnJrIBFmEFCluMBI=",
-        "windows_x86_64": "sha256-s25nQCGbmE1/JZ6ym0ZP54RiVI4GwYuyZ+Z7BfTu6qY=",
-    },
-    "v0.3.0": {
-        "darwin_x86_64": "sha256-+2KYkl/C+1yRG77SE29KaVn4qF00HBz1W7Zrxlm4xlI=",
-        "linux_arm": "sha256-e3UP5+6zo3IGM+ul/ev5EW7YevWzi7epErnH8/PIblQ=",
-        "linux_arm64": "sha256-PYQYMcTj5o0N8Hi3LIS6Vuyxoyfti4UR/hM+26OSBHY=",
-        "linux_i386": "sha256-PvGCXDdLMKDJwRDbiaw3+Ju1pINwIz5Yu7yC2FALRWs=",
-        "linux_x86_64": "sha256-KIAqfNQHxaSNNcRwue/aplVZWrf3DbciOEOSpPSwN38=",
-        "windows_x86_64": "sha256-BUMStVAV+8rLkOcNQihcKmh8FtiFw9YvGAUapH9AqUs=",
-    },
-    "v0.2.1": {
-        "darwin_x86_64": "sha256-AcyyNBeC+ehBieg9VokHbnKs8OxpRwdYj4yTlbiNPp8=",
-        "linux_arm": "sha256-4jZRHEylpKdDpWEzXPUYAsAYm5IR2SH8HdrWpnzONgQ=",
-        "linux_arm64": "sha256-m1wKFNy3TLHBdrY8LznKQRyGC/+p4XH1XEfzdnzVgJ0=",
-        "linux_i386": "sha256-Azl0K6gHxInfIJIbA8ePiGhZF1XJ33LbRrUywxxq/PA=",
-        "linux_x86_64": "sha256-sFpmBt2DdqvmkUAWkD9X+x/58aLLU9bPJejjYcEFG00=",
-        "windows_x86_64": "sha256-MuqImw41+kv/C8sw/4u49vJMfyge61fp4BFPBCtY5uk=",
-    },
-    "v0.2.0": {
-        "darwin_x86_64": "sha256-+Xri0Iiqhw2+IT3T3KQezPZjw3cfWrDwhiFloZ8p6HM=",
-        "linux_arm": "sha256-8rNIrS+/zkxLefHZlaOlJtrzCwxfaryg0LHiDOWJqb8=",
-        "linux_arm64": "sha256-CCDbFphv8RlRo873DHPkMfGtw4mWHcnMnNKzGnjtdQA=",
-        "linux_i386": "sha256-iITFRlNNvssZJ0rBgPKLE3QQQr0xOk3RTH7d7B4ODV4=",
-        "linux_x86_64": "sha256-mAlb+LUKbqUhc02gX+S8zc5gMesAQBlUiBf0eHIunt4=",
-        "windows_x86_64": "sha256-TUfhrBJdf0d1wJYsH/+UjP08SnKJREa5VfDgnBhx3L0=",
-    },
-    "v0.1.4": {
-        "darwin_x86_64": "sha256-43A0BONp7NssyOP/TCQRB2GKUpewJurPO/JRl1K/ZR0=",
-        "linux_i386": "sha256-bQ/Q0z9Op6zhng7p/UwSMOVXt7qsaUAiHqiH/i15bZE=",
-        "linux_x86_64": "sha256-0U81IVmWBXXCVRf7t6XmXF4ljUGJ8/vEq2mkigL5fv0=",
-    },
-    "v0.1.3": {
-        "darwin_x86_64": "sha256-8n4ZIIKmSXPkOhANbqe3OL1K+q+yAc5iRwGWPxcMIQI=",
-        "linux_i386": "sha256-ywrzfBkeW5Hy+UU08ebZH0s58+ZJlD5E0W5i8vZFXVk=",
-        "linux_x86_64": "sha256-BZ/Tl5VpLDMX2qk3i1fMaSF17Ky0Z6x7uUZXRbwrxvE=",
-    },
-    "v0.1.2": {
-        "darwin_i386": "sha256-UAndzfmBlw0KQ/ICat/R3nLDg1Pg8vyZB5V+8Jp0mBc=",
-        "darwin_x86_64": "sha256-aC9g1aXvjp1qx4iFp3Cl51qhkLL5qIlxufe/cP2clco=",
-        "linux_i386": "sha256-xfKuIyJLHEhk6uuv2rrDRyWBfT6LbWRS/upUyRXYqmE=",
-        "linux_x86_64": "sha256-dJ8j836a21s6HSVPJsMnMLKbbGwFkSoaZog4g67f/ks=",
-    },
-    "v0.1.1": {
-        "darwin_i386": "sha256-TPQtBirScb4qb3Y5+jSax8ynrcCcZRuWsxINsL6nf/c=",
-        "darwin_x86_64": "sha256-3GAnQ1LitQRSDpE1yv5LfvQMUsFWgWObtgEBFeuUqNQ=",
-        "linux_i386": "sha256-jEGfa0yYmGcYMLQAqjggrxslTHTOfLI6yRGKNm8Qfx8=",
-        "linux_x86_64": "sha256-ggpPUxrfME0wz3qFXy/eMg+9ADpxJf0zJUH4le9a4rU=",
-    },
+  "v0.12.0": {
```

Comment:
nit: comment about how this is the oldest version that's compatible with the rules, so we don't try to reintroduce earlier ones